pyomo/contrib/parmest/parmest.py

```diff
@@ -449,65 +393,46 @@ def _Q_opt(self, ThetaVals=None, solver="ef_ipopt",
         Set up all thetas as first stage Vars, return resulting theta
         values as well as the objective function value.
 
-        NOTE: If thetavals is present it will be attached to the
-        scenario tree so it can be used by the scenario creation
-        callback.  Side note (feb 2018, dlw): if you later decide to
-        construct the tree just once and reuse it, then remember to
-        remove thetavals from it when none is desired.
         """
-        
-        assert(solver != "k_aug" or ThetaVals == None)
-        # Create a tree with dummy scenarios (callback will supply when needed).
-        # Which names to use (i.e., numbers) depends on if it is for bootstrap.
+        if (solver == "k_aug"):
+            raise RuntimeError("k_aug no longer supported.")
```

Comment:
We might want to add an issue to support k_aug in the future now that we have easier access to it through `idaes-ext` in our testing environment.